### PR TITLE
Use https in production

### PR DESCRIPTION
**Pull request by peerwit**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -12,7 +12,11 @@ var mongoose = require('mongoose');
 var config = require('./server/config/environment');
 var https = require('https');
 var fs = require('fs');
-var credentials = {key: fs.readFileSync('./shared/config/ssl.key', 'utf-8'), cert: fs.readFileSync('./shared/config/ssl.crt', 'utf-8')};
+console.log(process.env.NODE_ENV);
+var credentials = {
+			key: process.env.NODE_ENV === 'production' ? fs.readFileSync('../shared/config/ssl.key', 'utf-8') : fs.readFileSync('./shared/config/ssl.key', 'utf-8'),
+			cert: process.env.NODE_ENV === 'production' ? fs.readFileSync('../shared/config/ssl.crt', 'utf-8') : fs.readFileSync('./shared/config/ssl.crt', 'utf-8')
```
Comment (kxprim) [inline server.js]: Indentation to fix

File: server.js [hunk 2]
```diff
@@ -28,22 +32,26 @@ var socketio = require('socket.io')(serverHTTPS, {
...
 });
 
-serverHTTP.listen(80, config.ip, function () {
-  // console.log(req.connection.encrypted?true:false);
-  console.log('Express server listening on %d, in %s mode', config.port, app.get('env'));
-});
+if (process.env.NODE_ENV === 'production') {
+	serverHTTP.listen(80, config.ip, function () {
+	  // console.log(req.connection.encrypted?true:false);
+	  console.log('Express server listening on %d, in %s mode', config.port, app.get('env'));
```
Comment (kxprim) [inline server.js]: Idem

**Comment by scottrice10** [comment]
:+1:

**Comment by kxprim** [comment]
:+1:

**Comment by rvbsanjose** [comment]
:+1:

**Pull request merged**